tests/KdybyTests/CsobPaymentGateway/Client.receiveResponse.phpt

```diff
@@ -46,6 +46,28 @@ class ClientReceiveResponseTest extends CsobTestCase
 
 
 
+	public function testReceiveResponseApproved()
+	{
+		$returnData = [
+				'payId' => 'fb425174783f9AK',
```

Comment:
chjo, to mi začal dělat phpstorm, když někam vkládám pole nebo i fluent volání, a nevím co s tím :(